spec/models/vm_or_template/operations/relocation_spec.rb

```diff
@@ -24,4 +24,10 @@
       )
     end
   end
+
+  context "supports" do
+    it "doesn't support migrate by default" do
```

Comment:
```suggestion
    it "does not support migrate by default" do
```
Let's not use contractions unless absolutely necessary